design/sketch.webidl

```diff
@@ -344,12 +346,12 @@ enum GPUCullMode {
 };
 
 dictionary GPURasterizationStateDescriptor {
-    GPUFrontFace frontFace;
-    GPUCullMode cullMode;
+    required GPUFrontFace frontFace;
+    GPUCullMode cullMode = "none";
```

Comment:
In most cases applications want to set a value different from none here.

Reply:
Then they'll set one. The default is mostly for "I don't care" users.

Reply:
Sounds good.